### ISSUE TITLE
VirtualMachineDiskOperationsImpl.updateDataDisk fail

### ISSUE THREAD

**Issue by chenrui1988** [Azure/azure-sdk-for-java#693]
I met  a issue when using VirtualMachineDiskOperationsImpl.updateDataDisk , I think It's maybe a bug. I use Service Manage API  in Azure SDK 0.9.3 

It always show me an error!
```
[Fatal Error] :1:1: Premature end of file.
com.microsoft.windowsazure.exception.ServiceException: 
	at com.microsoft.windowsazure.exception.ServiceException.createFromXml(ServiceException.java:216)
	at com.microsoft.windowsazure.management.compute.VirtualMachineDiskOperationsImpl.updateDataDisk(VirtualMachineDiskOperationsImpl.java:2219)
	at com.vianet.cie.vmdemo.CloudServiceAPITest.updateDataDisk(CloudServiceAPITest.java:83)
	at com.vianet.cie.vmdemo.CloudServiceAPITest.main(CloudServiceAPITest.java:53)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:144)
```
It's my code below:
```
VirtualMachineDiskGetResponse response = computeManagementClient.getVirtualMachineDisksOperations().getDisk("kevinvm-kevinvm-1-201605090139510196");
VirtualMachineDataDiskUpdateParameters updateParameters = new VirtualMachineDataDiskUpdateParameters();
updateParameters.setName("kevinvm-kevinvm-1-201605090139510196");
updateParameters.setLabel("kevinvm-data-1");
updateParameters.setMediaLinkUri(response.getMediaLinkUri());
updateParameters.setHostCaching(VirtualHardDiskHostCaching.READWRITE);
computeManagementClient.getVirtualMachineDisksOperations().updateDataDisk("kevinvm", "kevinvm", "kevinvm", 0, updateParameters);
```

I debuged it in my eclipse. and found this rest api return 202 status code. Then I checked it with the rest api document and found it return 202 code by design
![aa](https://cloud.githubusercontent.com/assets/4033321/15136311/e9ad32bc-16b2-11e6-8ef6-429c6aa2800a.jpg)
 
You could found this rest api in this link : https://msdn.microsoft.com/library/azure/jj157190.aspx
![bb](https://cloud.githubusercontent.com/assets/4033321/15136438/4130659e-16b4-11e6-86cd-488a3771f635.png)

**Comment by jianghaolu** [comment]
Yes it's a bug. Please take a look at the fix.

**Comment by chenrui1988** [comment]
Thanks,  Cloud you tell me when do you merge your patch to azure-sdk-for-java and when do you to publish a new version with this fix patch?

**Comment by jianghaolu** [comment]
This should be fixed in 0.9.5.
